js/views/pageNavVw.js

```diff
@@ -356,6 +358,27 @@ module.exports = baseVw.extend({
     app.hideOverlay();
     this.$notifMenu.removeClass('popMenu-opened');
   },
+  
+  markNotificationAsRead: function(view) {
```

Comment:
You're welcome. 

I just didn't think clicking on a single notification should need to mark all others as read. But because you get a meaning of the notification right from the menu, it kind of makes sense to close all of them at once.